### ISSUE TITLE
Running injectived with --with-tendermint=false does not work

### ISSUE THREAD

**Issue by troykessler** [InjectiveFoundation/injective-core#4]
For the latest injectived version `v1.12.1` running injectived in normal mode (`./injectived start`) does work, however trying to start injectived without tendermint (in order to run tendermint in a separate process) starts injectived in the normal mode, therefore ignoring the flag `--with-tendermint=false`. Checking the injective fork of the cosmos-sdk `v0.47.3-inj-9` which is used in `v1.12.1` it apparently also provides the `--with-comet` flag, which unfortunately errors with `Error: unknown flag: --with-comet`

**Comment by achilleas-kal** [comment]
This should be fixed in v1.13

**Comment by troykessler** [comment]
> This should be fixed in v1.13

Just tried out v1.13.0 and v1.13.1 and getting the following error now:

```
9:45AM INF starting ABCI without CometBFT module=main
9:45AM ERR Running without CometBFT is not supported. module=main
9:45AM INF starting node with ABCI CometBFT in-process module=main
```

So apparently it registers that the flag was provided with the value false, but refuses to start without Cometbft in-process

**Comment by achilleas-kal** [comment]
Hey @troykessler I'll relay this to the team, thanks!
